### ISSUE TITLE
Jet Appointment - lead time 

### ISSUE THREAD

**Issue by levi-hrb** [CrocoBlock/suggestions#2917]
It would be super helpful to have an option in Jet Appointment to set x days as a lead time. For example a service needs 4 days of preparation or the provider just has to know about the job 4 days earlier, it would be helpful to set a parameter to every service on how many days are needed to prepare or how much "lead time" is needed.  Then the user would not be able to select today and the next three days, but after 4 days, that would give the provider always enough time to prepare.
Thank you for your job!

**Comment by rich73625** [comment]
Yes, in a clinic setting if there are no bookings then the Provider may not need to go in until afternoon, for example. Setting a lead time would mean that a client can't book up a session which starts in 30 minutes, and then just turn up, without giving the provider time to get there/prepare etc.
